js/ui/widget/ui.widget.js

```diff
@@ -262,64 +245,52 @@ var Widget = DOMComponentWithTemplate.inherit({
         this._attachHoverEvents();
     },
 
-    _renderHint: function() {
-        var hint = this.option("hint");
-        this.$element().attr("title", hint ? hint : null);
+    _renderHint() {
+        const { hint } = this.option();
+
+        this.$element().attr('title', hint || null);
     },
 
-    _renderContent: function() {
-        commonUtils.deferRender(() => {
-            if(this._disposed) {
-                return;
-            }
-            return this._renderContentImpl();
-        }).done(() => {
-            if(this._disposed) {
-                return;
-            }
-            this._fireContentReadyAction();
-        });
+    _renderContent() {
+        deferRender(() => !this._disposed ? this._renderContentImpl() : void 0)
```

Comment:
```suggestion
        deferRender(() => !this._disposed ? this._renderContentImpl() : undefined)
```
As I got it, this should preserve the global `undefined` variable from been overwritten in some ancient browser. If we think this can a problem we should better create a dedicated PR for updating all such places in devextreme.

Actually it seems to be non-relevant for our days)

Reply:
I like to use `void 0`. Not so important but looks better for me. I can rollback it if you want.

Reply:
I think we could add this to the next guide-lines-meeting agenda